src/autocapture-utils.ts

```diff
@@ -70,7 +70,7 @@ export function getSafeText(el: Element): string {
  * @param {Element} el - element to check
  * @returns {boolean} whether el is of the correct nodeType
  */
-export function isElementNode(el: Element | undefined | null): el is HTMLElement {
+export function isElementNode(el: Node | Element | undefined | null): el is Element {
```

Comment:
Should it be `Node` only instead?
https://developer.mozilla.org/en-US/docs/Web/API/Node
Element implements `Node`

Reply:
So, this is a check to narrow from Node to Element. So we can do things like rely on `tagName` being present

Reply:
It was previously asserting the type could be considered HTMLElement after checking node type === 1, but that's not true